src/meta/src/hummock/manager/gc.rs

```diff
@@ -175,44 +173,36 @@ impl GcManager {
 }
 
 impl HummockManager {
-    /// Deletes at most `batch_size` deltas.
+    /// Deletes version deltas.
     ///
-    /// Returns (number of deleted deltas, number of remain `deltas_to_delete`).
-    pub async fn delete_version_deltas(&self, batch_size: usize) -> Result<(usize, usize)> {
+    /// Returns number of deleted deltas
+    pub async fn delete_version_deltas(&self) -> Result<usize> {
         let mut versioning_guard = self.versioning.write().await;
         let versioning = versioning_guard.deref_mut();
         let context_info = self.context_info.read().await;
-        let deltas_to_delete_count = versioning
-            .hummock_version_deltas
-            .range(..=versioning.checkpoint.version.id)
-            .count();
         // If there is any safe point, skip this to ensure meta backup has required delta logs to
         // replay version.
         if !context_info.version_safe_points.is_empty() {
-            return Ok((0, deltas_to_delete_count));
+            return Ok(0);
         }
```

Comment:
An implicit problem may cause the request to time out if the number of bulk deletions is too large.

Optional Handling Means
- Stop the service
- Log in to pg and actively clean it up.

Reply:
just for the record, as we think its probability is low.